upstream-commit.txt

```diff
@@ -1 +1 @@
-cf808a16bff0eb531a48debcd04be0aab562c21e
+3d97ee77ff70ab1093a63e4d3dadda9e885fe857
```

Comment:
question: maybe I am missing something but why is this commit different from the one you are back porting?

Reply:
Because there are two merge commits in snakeyaml-engine after the one I'm porting, they both seem to deal with this commit with the comment fix. This hash points to the second merge commit. See https://raw.githubusercontent.com/krzema12/snakeyaml-engine-kmp/refs/heads/commits-to-upstream-badge/log-diff-between-repos.txt:

```
3d97ee7 Merge remote-tracking branch 'refs/remotes/github/master'
91fd294 Merge pull request #2 from aSemy/patch-1
e274d60 Update setEnvConfig Javadoc in LoadSettingsBuilder
```

Reply:
Got it, thanks for the explanation